Gemfile.lock

```diff
@@ -215,3 +216,6 @@ DEPENDENCIES
   standard
   toxiproxy
   yalphabetize
+
+BUNDLED WITH
+   2.3.7
```

Comment:
You somehow keep merging this in into this branch by mistake 🙈

Reply:
This is my biggest gripe with Bundler. This insistence on using a precise version. It should have just been major version. Removed the .lock file.